persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/global/RemoteContainerConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.persistence.remote.configuration.global;
+
+import static org.infinispan.commons.configuration.AbstractTypedPropertiesConfiguration.PROPERTIES;
+
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.commons.configuration.attributes.ConfigurationElement;
+import org.infinispan.commons.util.Immutables;
+import org.infinispan.commons.util.TypedProperties;
+import org.infinispan.persistence.remote.configuration.Attribute;
+import org.infinispan.persistence.remote.configuration.Element;
+
+/**
+ * @since 15.0
+ **/
+public class RemoteContainerConfiguration extends ConfigurationElement<RemoteContainerConfiguration> {
+   static final AttributeDefinition<String> NAME = AttributeDefinition.builder(Attribute.NAME, "", String.class).immutable().build();
+   static final AttributeDefinition<String> URI = AttributeDefinition.builder(Attribute.URI, null, String.class).immutable().build();
+   private final org.infinispan.commons.configuration.attributes.Attribute<TypedProperties> properties;
+
+   protected RemoteContainerConfiguration(AttributeSet attributes, ConfigurationElement<?>... children) {
+      super(Element.REMOTE_CACHE_CONTAINER, attributes, children);
+      this.properties = attributes.attribute(PROPERTIES);
```

Comment:
Are we actually using properties here? Seems unneeded, no?

Reply:
Were you able to confirm if the properties here are required?

Reply:
I tried allowing the same elements the `remote-store` accepts, but that would be a lot of duplication or changing the existing configuration. As is, we can't use the same builders for the other elements, as they are coupled with the store configuration.

I think we could update it to accept all the same configuration options in other PR/JIRA.